dotcom-rendering/src/components/Card/components/CardLayout.tsx

```diff
@@ -128,16 +129,16 @@ const decidePosition = (
 };
 
 /** Detemines the gap size between components in card layout */
-const decideGap = (gapSize: GapSize) => {
+const decideGap = (gapSize: GapSize, isBetaContainer: boolean) => {
```

Comment:
Instead of updating fixing the sizing here, could we update the getGapSize function to pass through `tiny` as the gap size prop (which is 4px)?